apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -17,10 +17,11 @@ $(document).ready(function () {
   ReactDOM.render(
     <Provider store={store}>
       <Congrats
-        completedTutorialType="other"
+        completedTutorialType="2017Minecraft"
         isRtl={isRtl}
         userType={userType}
         isEnglish={isEnglish}
+        MCShareLink="minecraft/sharelink"
```

Comment:
These are being set here to test that the MCShareLink is passed as expected.  Ultimately, completedTutorialType and MCShareLink will be pulled from the completed tutorials url.